### PR TITLE
Add AgentRingNoAnswerEvent

### LINKED ISSUE #165: AgentRingNoAnswer Event missing
Hi guys

Please also add the AgentRingNoAnswer event class.
That might be useful to some users.

Regards!

### PR DESCRIPTION
**Pull request by moldypenguins**
Adds AgentRingNoAnswerEvent.

Fixes #165

### CODE REVIEW

**Comment by moldypenguins** [comment]
See issue #165 AgentRingNoAnswer Event missing

File: Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs [hunk 1]
```diff
@@ -120,6 +120,11 @@ public class ManagerConnection
         /// </summary>
         public event EventHandler<AgentLogoffEvent> AgentLogoff;
         /// <summary>
+        /// An AgentRingNoAnswer is triggered when an agent was rang and did not answer.<br/>
+        /// To enable AgentRingNoAnswer you have to set eventwhencalled = yes in queues.conf.<br/>
```
Comment (Deantwo) [inline Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs]: A `<br/>` at the end of the text make no sense.

File: Asterisk.2013/Asterisk.NET/Manager/Event/AgentRingNoAnswerEvent.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace AsterNET.Manager.Event
+{
+    /// <summary>
+    ///     Raised when a queue member is notified of a caller in the queue and fails to answer.
+    ///
```
Comment (Deantwo) [inline Asterisk.2013/Asterisk.NET/Manager/Event/AgentRingNoAnswerEvent.cs]: I don't think this works as a linebreak. You have to use `<br/>` instead.

File: Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs (hunk 1 above)
Comment (moldypenguins) [reply Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs]: those were there already

File: Asterisk.2013/Asterisk.NET/Manager/Event/AgentRingNoAnswerEvent.cs (hunk 2 above)
Comment (moldypenguins) [reply Asterisk.2013/Asterisk.NET/Manager/Event/AgentRingNoAnswerEvent.cs]: more for readability in comments, i can remove the extra whitespace if wanted?

File: Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs (hunk 1 above)
Comment (moldypenguins) [reply Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs]: oh i guess they weren't, maybe they copied over with the text? either way, removed them

**Comment by moldypenguins** [comment]
Should be fixed now

File: Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs [hunk 3]
```diff
@@ -120,8 +120,8 @@ public class ManagerConnection
         /// </summary>
         public event EventHandler<AgentLogoffEvent> AgentLogoff;
         /// <summary>
-        /// An AgentRingNoAnswer is triggered when an agent was rang and did not answer.<br/>
```
[the remark this answers is not in the document]
Comment (moldypenguins) [reply Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs]: my bad, added back

**Comment by skrusty** [comment]
we happy to merge this? @Deantwo

**Pull request merged**